Common/UnitDefinitions/VascularResistance.json

```diff
@@ -0,0 +1,76 @@
+﻿{
+  "Name": "VascularResistance",
+  "BaseUnit": "DyneSecondPerCentimeterToTheFifth",
+  "XmlDocSummary": "Vascular resistance is the resistance that must be overcome for blood to flow through the circulatory system.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Vascular_resistance",
```

Comment:
This quantity on the surface feels very domain specific, and I am a little hesitant to add it as we strive to balance adding the most widely used units that many benefit from, without bloating the library with exotic units. There are extremely many domains in the world, with their own units. 

Could this be generalized into fluid resistance maybe? It seems to have the same dimensions of the units at least.

https://www.vcalc.com/wiki/Fluid-Resistance-Unit-Conversion

Reply:
Sounds logical, I'll add it sometime after the holidays.